pkg/console/controllers/util/informers_test.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+// func TestEnsureRunningAndStop(t *testing.T) {
```

Comment:
I expect this will get uncommented ?

Reply:
Yeah I had commented it out while the test was broken; this has now been added back in.